packages/react/src/components/Avatar/Avatar.tsx

```diff
@@ -116,10 +116,6 @@ class Avatar extends UIComponent<ReactProps<AvatarProps>, any> {
           defaultProps: {
             size,
             styles: styles.status,
-            variables: {
-              borderColor: variables.statusBorderColor,
-              borderWidth: variables.statusBorderWidth,
-            },
```

Comment:
The status itself no longer needs these variables.  These are only used in the context of the Avatar.  So, the Avatar adds styles to the Status component in this case.